ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
```

Comment:
Why are these comments here? Shouldn't we just define these keys here to some value (i.e., empty string, or None?) so that they always exist? Currently, if I run `python server_dev.py -c LocalOidConfig` I get a key error after the first request. This is very poor behaviour: if we are missing configuration variables, these should be reported at start-time, not when a request is made/